study.html

```diff
@@ -0,0 +1,469 @@
+            <img src="assets/img/calgraf.png" class="img-responsive" alt="Календарный учебный график" />
+          </figure>
+
+          <div class="columns">
+            <div class="col-7 col-md-12">
+              <div class="tile tile-centered">
+                <div class="tile-icon">
+                  <i class="icon icon-insert_drive_file centered"></i>
+                </div>
+                <div class="tile-content">
+                  <div class="tile-title">Годовой календарный учебный график на 2016/17 учебный год</div>
+                  <div class="tile-subtitle">14 МБ <b>·</b> ODT-файл <b>·</b> 01 янв 2017</div>
+                </div>
+                <div class="tile-action">
+                  <a href="#" class="btn btn-link">
+                    <i class="icon icon-get_app"></i>
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+          <section class="ml-10">
+            <h6>Расписание звонков</h6>
+            <div class="columns">
+              <div class="col-4 col-md-8">
+                <table class="table text-center">
+                  <thead>
+                    <tr>
+                      <th></th>
+                      <th>Начало</th>
+                      <th>Окончание</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    <tr>
+                      <td>1 урок</td>
+                      <td>9:00</td>
+                      <td>9:40</td>
+                    </tr>
+                    <tr>
+                      <td>2 урок</td>
+                      <td>9:50</td>
+                      <td>10:30</td>
+                    </tr>
+                    <tr>
+                      <td>3 урок</td>
+                      <td>10:45</td>
+                      <td>11:25</td>
+                    </tr>
+                    <tr>
+                      <td>4 урок</td>
+                      <td>11:40</td>
+                      <td>12:20</td>
+                    </tr>
+                    <tr>
+                      <td>5 урок</td>
+                      <td>12:35</td>
+                      <td>13:15</td>
+                    </tr>
+                    <tr>
+                      <td>6 урок</td>
+                      <td>13:35</td>
+                      <td>14:15</td>
+                    </tr>
+                    <tr>
+                      <td>7 урок</td>
+                      <td>14:35</td>
+                      <td>15:15</td>
+                    </tr>
+                  </tbody>
+                </table>
+              </div>
+            </div>
+          </section>
+        </article>
+
+        <div class="anchor" id="plan"></div>
+        <article>
+          <h3>Учебные планы и программы</h3>
+          <div class="divider"></div>
+          <ul>
+            <li>
+              <button class="btn btn-link">
+                <i class="icon icon-arrow_drop_down"></i>
```

Comment:
It would be better to have `keyboard_arrow_right` while toggled off and then switching to `keyboard_arrow_down`, wouldn't it?
Btw it seems that all panes should be closed by default.